tests/test_price_deviation_coingecko.py

```diff
@@ -0,0 +1,48 @@
+from pyth_observer.events import PriceDeviation, PriceDeviationCoinGecko
+from pythclient.pythaccounts import PythPriceInfo, PythPriceStatus
+from pyth_observer.prices import Price
+
+
+class MockPythProductAccount:
```

Comment:
Have you seen [unittest.mock](https://docs.python.org/3/library/unittest.mock.html)? I've found this super useful for easily mocking python objects with minimal effort/boilerplate.

Reply:
yup I used that for https://github.com/pyth-network/pyth-client-py -- was trying to follow the existing tests which was using a mock class, happy to migrate all to unittest in a follow up PR